src/frontend.jl

```diff
@@ -54,17 +54,22 @@ macro progress(ex)
 end
 
 """
-    progress(x = [0..1])
+    progress(x = [0..1], [message])
 
-Set Atom's progress bar to the given value.
+Set Atom's progress bar to the given value. Displays `message` as a tooltip.
 """
 progress(x::Void = nothing) = Atom.msg("progress", "indeterminate")
 
 progress(x::Real) =
-  Atom.msg("progress", (x < 0.01 ? nothing :
+  Atom.msg("progress", (x < 0.01 ? "indeterminate" :
                         x > 1 ? 1 :
                         x))
 
+progress(x::Real, message::String) =
+  Atom.msg("progress", (x < 0.01 ? "indeterminate" :
```

Comment:
Will this write "indeterminate" instead of the message whenever `x<0.01`? I think it would be good to still show the message. I think this message would be a good place to show `verbose` information, like I would like to say "Initial dt: ####" at the start, and other information.